### PR TITLE
esbuild cache invalidation: add argv to optionsHash

### PR DESCRIPTION
**Pull request by samouri**
Alternative to https://github.com/ampproject/amphtml/pull/33365#event-4478137702

### CODE REVIEW

File: build-system/common/esbuild-babel.js [hunk 1]
```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
  */
 
+const argv = require('minimist')(process.argv.slice(2));
```
Comment (jridgewell) [inline build-system/common/esbuild-babel.js:17]: Nit: no need to process it, just get the raw array.
```suggestion
const argv = process.argv.slice(2);
```

**Review by jridgewell — APPROVED** [review]

File: build-system/common/esbuild-babel.js (hunk 1 above)
Comment (rsimha) [reply build-system/common/esbuild-babel.js:17]: Or remove this `require` and directly use `process.argv` in `optionsHash`?

Comment (samouri) [reply build-system/common/esbuild-babel.js:17]: both ideas work

**Comment by rsimha** [comment]
Yay! This worked: 
image

**Pull request merged**